django/contrib/admindocs/views.py

```diff
@@ -60,11 +55,11 @@ class TemplateTagIndexView(BaseAdminDocsView):
     template_name = 'admin_doc/template_tag_index.html'
 
     def get_context_data(self, **kwargs):
-        load_all_installed_template_libraries()
-
+        # Non-trivial TEMPLATES settings aren't supported (#24125).
+        engine = Engine.get_default()
```

Comment:
Yes, it can be reached. I figured it should fail loudly, but I see elsewhere that ``admindocs`` silences the error. I'll update that and leave a better fix for ticket 24125.